core/server/helm_release_test.go

```diff
@@ -0,0 +1,109 @@
+package server
```

Comment:
can we make this `server_test`? we are testing an exposed interface, so we can test against it from the outside

Reply:
I'm 100% on the that side (blackbox testing), but the rest of the code follows this logic (whitebox testing), so I did the same.

Reply:
Did a few poke on it, all the tests are using the same functions (and global variable), so I moved everything and now it's all `server_test`